src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -1786,15 +1787,16 @@ private static void ensureDetailedGHPullRequest(
       GHPullRequest pr, TaskListener listener, GitHub github, GHRepository ghRepository)
       throws IOException, InterruptedException {
     final long sleep = 1000;
-    int retryCountdown = 4;
+    int retryCountdown =
+        SystemProperties.getInteger(
+            GitHubSCMSource.class.getName() + ".mergeableStatusRetries", Integer.valueOf(4));
```

Comment:
This is a complete nit, but don't we usually add these as static constants? i.e `MERGEABLE_STATUS_RETRIES`?

Reply:
fixed in edfa92692b303f9507d26a02d7ced3121c9dad92